File: Excel_Programs/Python/question7.py
import numpy as np
n = int(input("Enter the number of elements to be entered : "))
arr = []
for i in range(n):
    arr.append(int(input("Enter a number : ")))
arr=np.array(arr)
temp=0
for i in range(n):
    temp= arr[i];
    arr[i]=arr[n-1];
    arr[n-1]= temp;
print(arr)

File: Excel_Programs/C++/question7.cpp
#include<iostream>
using namespace std;

int main()
{
    int n;
    cout<<"Enter the number of elements to be entered : ";
    cin>>n;
    int arr[n];
    for(int i=0;i<n;i++)
    {
        cout<<"Enter a number : ";
        cin>>arr[i];
    }
    int temp;
    for(int i=0,j=n-1;i<n;i++)
    {
        temp= arr[i];
        arr[i]=arr[j];
        arr[j]= temp;
        for(int i =0; i<n;i++)
        {
            cout<<arr[i]<<" ";
        }
        cout<<"\n";
    }
    cout<<"\n";
    for(int i =0; i<n;i++)
    {
        cout<<arr[i]<<" ";
    }
}